marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/base.xqy

```diff
@@ -247,42 +247,49 @@ declare function merge-impl:construct-type($name as xs:QName, $path as xs:string
  :)
 declare function merge-impl:generate-audit-attachments(
```

Comment:
Refactored this function to avoid duplicate work done by the provenance details map created for Data Hub 5 provenance.